embassy-net-adin1110/src/lib.rs

```diff
@@ -0,0 +1,1249 @@
+#![allow(clippy::pedantic)]
+#![feature(type_alias_impl_trait)]
+#![feature(async_fn_in_trait)]
+#![cfg_attr(not(any(test, feature = "std")), no_std)]
```

Comment:
can you add `#![doc = include_str!("../README.md")]` so that the README shows up in the main page of rustdocs?